com.yakindu.sct.examples.coffeemachine.c/src/coffee_machine.h

```diff
@@ -11,7 +11,7 @@
 #include <stdbool.h>
 #include <stdint.h>
 
-#include "cm_hmi.h"
```

Comment:
this is strange because the header is already in the src folder, so that should not be necessary

Reply:
Yep, but the code does not compile

We can push this and remove it later on. But without this fix, the example won't run on windows.